lib/usd/translators/mayaReferenceReader.cpp

```diff
@@ -52,8 +52,19 @@ PXRUSDMAYA_DEFINE_READER(MayaUsd_SchemasMayaReference, args, context)
 
     MObject referenceParentNode;
 
-    UsdMayaTranslatorUtil::CreateTransformNode(
-        usdPrim, parentNode, args, &context, &status, &referenceParentNode);
+    // The stand-in for a Maya reference prim is a dummy transform node, which
+    // preserves the source USD prim type in attribute "USD_typeName".  To
+    // support workflows where this dummy transform node has its transform
+    // changed, we leave its transform unlocked.
+    UsdMayaTranslatorUtil::CreateDummyTransformNode(
```

Comment:
Important change: translating a Maya reference prim now creates a dummy transform node, which is USD type-preserving.  Creating a plain Maya transform node does not preserve USD type information, which makes pull / push round-tripping very difficult, since we wish to use the same updater for push as was used for pull.